lib/cc/engine/file_list_resolver.rb

```diff
@@ -29,13 +29,13 @@ def exclude_due_to_config?(path)
       end
 
       def include_based_files_to_inspect
-        @include_paths.map do |path|
-          if path =~ %r{/$}
+        @include_paths.map(&method(:abs_path)).flat_map do |path|
```

Comment:
I meant adding a method to hide both `@include_paths` and what you're doing above with the method proc.